src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -768,6 +817,8 @@ public Map<String, Object> convertToMapOfObjects() {
             currentProcessingFileInformationMap.put(PERSISTED_CURRENT_PROCESSING_FILE_START_POSITION, startPosition);
             currentProcessingFileInformationMap.put(PERSISTED_CURRENT_PROCESSING_FILE_LAST_MODIFIED_TIME,
                     lastModifiedTime);
+            currentProcessingFileInformationMap.put(PERSISTED_CURRENT_PROCESSING_FILE_LAST_ACCESSED_TIME,
```

Comment:
Store the lastAccessed time here in case the customer restarts greengrass or the LM component